ddl.dic

```diff
@@ -546,6 +546,24 @@ save_dictionary.ddl_conformance
 
 save_
 
+save_dictionary.doi
+
+    _definition.id                '_dictionary.doi'
```

Comment:
```suggestion
    _definition.id                '_dictionary.DOI'
```

Reply:
The capitalised version of the DOI acronym seems to be used in data names throughout CIF_CORE.